tests/requirements.txt

```diff
@@ -1,6 +1,22 @@
-matplotlib
-mypy==0.950
+# Tools used for testing
+docopt-ng
```

Comment:
When installing the requirements on a fresh install of Python 3.10, I got the following warning:
`  DEPRECATION: docopt is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559`

That led me to https://github.com/docopt/docopt/issues/502#issuecomment-1289347288: even docopt organization members are recommending to switch over to docopt-ng due to docopt being unmaintained in the past 6 years (last release 10 years ago)